concrete/src/Attribute/Controller.php

```diff
@@ -431,6 +441,8 @@ public function getLabelID()
     }
 
     /**
+     * @deprecated . This should be handled by the templates including <label> tags and using
```

Comment:
Since this is `@deprecated` the business logic it contains should be moved to non-deprecated methods. Ideally we shouldn't ever have a reason to call a deprecated method in the core since the business logic will all be elsewhere anyway.